root/main/error/components/ErrorInfo.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * @flow strict
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+type Props = {
+  +formattedErrors?: $ReadOnlyArray<string>,
+  +message?: string,
+};
+
+const ErrorInfo = ({
```

Comment:
Same here, I just found it comes from `/root/main/info/error.tt` but commit https://github.com/metabrainz/musicbrainz-server/pull/1699/commits/b5ef9f12c395ef4135313c62470437d5fdb48c08 gave no indication.